example/sysinfo/odp_sysinfo.c

```diff
@@ -1073,6 +1079,31 @@ int main(int argc, char **argv)
 	printf("    max_put_batch:        %u\n", stash_capa.max_put_batch);
 	printf("    stats:                0x%" PRIx64 "\n", stash_capa.stats.all);
 
+	printf("\n");
+	printf("  ML\n");
+	printf("    max_models:             %u\n", ml_capa.max_models);
+	printf("    max_models_loaded:      %u\n", ml_capa.max_models_loaded);
+	printf("    max_model_size:         %" PRIu64 "B\n", ml_capa.max_model_size);
+	printf("    max_compl_id:           %u\n", ml_capa.max_compl_id);
+	printf("    max_inputs:             %u\n", ml_capa.max_inputs);
+	printf("    max_outputs:            %u\n", ml_capa.max_outputs);
+	printf("    max_segs_per_input:     %u\n", ml_capa.max_segs_per_input);
+	printf("    max_segs_per_output:    %u\n", ml_capa.max_segs_per_output);
+	printf("    min_input_align:        %u\n", ml_capa.min_input_align);
+	printf("    min_output_align:       %u\n", ml_capa.min_output_align);
```

Comment:
Prints are missing for `packed_input_data` and `packed_output_data` capabilities.